conbench/app/benchmarks.py

```diff
@@ -158,6 +158,8 @@ def _augment(self, run):
         # display_message really seems to be name of the link
         run["commit"]["display_message"] = commit_message
         if not commit_message:  # None or empty string
+            # Note(JP): I wonder where the "url" ever got set.
```

Comment:
/shrug I have no idea what commit url is supposed to be. This is probably dead code

Reply:
:+1: 

By now I think I know this comes from here: https://github.com/conbench/conbench/blob/fa5663866a753d8c2e22ec66d5477aca8931ca2c/conbench/entities/commit.py#L241

i.e. this challenging to answer because of an unnecessary indirection, the API layer re-serialization plus augmentation, analogue to what's described in https://github.com/conbench/conbench/issues/890#issue-1629055001.